quipucords/api/source/tests_source.py

```diff
@@ -1035,16 +1091,25 @@ def test_create_sat_with_hosts(self):
              'hosts': ['1.2.3.4', '1.2.3.5'],
              'credentials': [self.sat_cred_for_upload]})
 
+    def test_create_sat_with_exclude_hosts(self):
+        """A satellite source does not accept excluded hosts."""
+        self.create_expect_400(
+            {'name': 'source1',
+             'source_type': Source.SATELLITE_SOURCE_TYPE,
+             'hosts': ['1.2.3.4'],
+             'exclude_hosts': ['1.2.3.4'],
+             'credentials': [self.sat_cred_for_upload]})
+
     def test_create_sat_with_host_range(self):
-        """A vcenter source must not have multiple hosts."""
+        """A satellite source must not have multiple hosts."""
         self.create_expect_400(
             {'name': 'source1',
              'source_type': Source.SATELLITE_SOURCE_TYPE,
              'hosts': ['1.2.3.4/5'],
              'credentials': [self.sat_cred_for_upload]})
 
     def test_create_req_type(self):
-        """A vcenter source must have an type."""
+        """A satellite source must have an type."""
```

Comment:
I think that this one would actually be a vcenter source, since the cred being used is vcenter.

Reply:
It was among all the other satellite tests, I'll move it to the vcenter section